TwitchLib.Api.Helix.Models/Schedule/Segment.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace TwitchLib.Api.Helix.Models.Schedule
+{
+    public class Segment
+    {
+        [JsonProperty("id")]
+        public string Id { get; protected set; }
+        [JsonProperty("start_time")]
+        public string StartTime { get; protected set; }
+        [JsonProperty("end_time")]
+        public string EndTime { get; protected set; }
+        [JsonProperty("title")]
+        public string Title { get; protected set; }
+        [JsonProperty("canceled_until")]
+        public string CanceledUntil { get; protected set; }
```

Comment:
```
Used with recurring scheduled broadcasts. Specifies the date of the next recurring broadcast in RFC3339 format if one or more specific broadcasts have been deleted in the series. Set to null otherwise.
```
We can use nullable DateTIme I think here?